recipes/pyramid/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "pyramid" %}
+{% set version = "1.10.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 37c3e1c9eae72817e0365e2a38143543aee8b75240701fa5cb3a1be86c01a1c0
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pcreate=pyramid.scripts.pcreate:main
+    - pserve=pyramid.scripts.pserve:main
+    - pshell=pyramid.scripts.pshell:main
+    - proutes=pyramid.scripts.proutes:main
+    - pviews=pyramid.scripts.pviews:main
+    - ptweens=pyramid.scripts.ptweens:main
+    - prequest=pyramid.scripts.prequest:main
+    - pdistreport=pyramid.scripts.pdistreport:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - hupper
+    - pip
+    - plaster
+    - plaster_pastedeploy
+    - python
+    - setuptools
+    - translationstring >=0.4
+    - venusian >=1.0
+    - webob >=1.8.3
+    - zope.deprecation >=3.5.0
+    - zope.interface >=3.8.0
+  run:
+    - hupper
+    - plaster
+    - plaster_pastedeploy
+    - python
+    - setuptools
+    - translationstring >=0.4
+    - venusian >=1.0
+    - webob >=1.8.3
+    - zope.deprecation >=3.5.0
+    - zope.interface >=3.8.0
+
+test:
+  imports:
+    - pyramid
+    - pyramid.config
+    - pyramid.scaffolds
+    - pyramid.scripts
+  commands:
+    - pcreate --help
+    - pserve --help
+    - pshell --help
+    - proutes --help
+    - pviews --help
+    - ptweens --help
+    - prequest --help
+    - pdistreport --help
+  requires:
+    - webtest >=1.3.1
+    # - zope.component >=4.0
```

Comment:
Why is this commented? Please leave a comment with a reason. 👍

Reply:
Put a better explanation.